.drone.star

```diff
@@ -612,6 +613,42 @@ def binary(ctx, name):
     },
   }
 
+def updateDeployment(ctx):
+  return {
+    'kind': 'pipeline',
+    'type': 'docker',
+    'name': 'updateDeployment',
+    'platform': {
+      'os': 'linux',
+      'arch': 'amd64',
+    },
+    'steps': [
+      {
+        'name': 'webhook',
+        'image': 'plugins/webhook',
+        'settings': {
+          'username': {
+            'from_secret': 'webhook_username',
+          },
+          'password': {
+            'from_secret': 'webhook_password',
+          },
+          'method': 'GET',
+          'urls': 'https://ocis.owncloud.works/hooks/update-ocis',
+        }
+      }
+    ],
+    'depends_on': [
```

Comment:
please add more deps here.
We need to trigger only if all docker pipelines were successful

Reply:
Is that so? Because our instance is an amd64 Linux machine. But sure it doesn't hurt to have all other steps finish first.

Reply:
yes, correct, but we should't deploy if we have errrors on other platforms. We should investiate.
Agreed?

Reply:
True. :+1: